Updated the UI references (#2234)
influxdata/docs-v2: content/influxdb/cloud/organizations/buckets/update-bucket.md

## Update a bucket's name in the InfluxDB UI

1. In the navigation menu on the left, select **Data ** > **Buckets**.

>     {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}

Reviewer: This is a convention we've established to address both the collapsed and expanded states of the left nav. These should not be changed.

>     {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}

Reviewer: ```suggestion
    {{< nav-icon "data" >}}
```

> 2. Click **Settings** to the right of the current name of the bucket that you want to rename.

Reviewer: ```suggestion
2. Click **Settings** to the right of the bucket you want to rename.
```

3. Click **Rename**.
3. Review the information in the window that appears and click **I understand, let's rename my bucket**.
4. Update the bucket's name and click **Change Bucket Name**.

## Update a bucket's retention policy in the InfluxDB UI

1. In the navigation menu on the left, select **Data ** > **Buckets**.

    {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}

>     

Reviewer: This is a convention we've established to address both the collapsed and expanded states of the left nav. These should not be changed.

> 1. In the navigation menu on the left, select **Data ** > **Buckets**.
> 
>     {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}
>     

Reviewer: ```suggestion
1. In the navigation menu on the left, select **Data (Load Data)** > **Buckets**.

    {{< nav-icon "data" >}}
    
```

2. Click **Settings** next to the bucket you want to update.

> 3. In the window that appears, select from the predefined time ranges for the bucket's retention policy. Note: You can use the CLI commands further below on this page to set the retention policy to different values not shown in the UI.
>     {{![image](https://user-images.githubusercontent.com/9125473/109828711-7fbfff00-7c02-11eb-94fa-9acc07a65296.png)}}

Reviewer: ```suggestion
3. In the window that appears, under **Delete data**, select a retention period:

    - **Never**: data in the bucket is retained indefinitely.
    - **Older Than**: select a predefined retention period from the dropdown menu.
    
    {{% note %}}
Use the [`influx bucket update` command](#update-a-buckets-retention-policy) to set a custom retention policy.
    {{% /note %}}
```
